src/group/managed_group/ser.rs

```diff
@@ -0,0 +1,51 @@
+use crate::group::managed_group::*;
+
+use serde::{
+    ser::{SerializeStruct, Serializer},
+    Deserialize, Serialize,
+};
+#[derive(Serialize, Deserialize)]
+pub struct SerializedManagedGroup {
+    managed_group_config: ManagedGroupConfig,
+    group: MlsGroup,
+    pending_proposals: Vec<MLSPlaintext>,
+    own_kpbs: Vec<KeyPackageBundle>,
+    aad: Vec<u8>,
+    active: bool,
+}
+
+impl<'a> SerializedManagedGroup {
+    pub(crate) fn into_managed_group(
+        self,
+        credential_bundle: &'a CredentialBundle,
```

Comment:
Why is the credential bundle not part of the serialized group? Is this in preparation for #214?

Reply:
Yes. We don't want to serialize the signing in every group.